internal/builder/testdata/existing.yml

```
@@ -7,7 +7,9 @@ elasticsearch:
         example:
           type: short
       dynamic_templates:
-        - to_nested:
-            path: trigger
+        - forwarded_ip_to_ip:
             mapping:
-              type: nested
+                type: ip
```

I think CI complains about some extra space here.
```suggestion
              type: ip
```